Client/Experiments/Messaging/GleanPlumbMessageManager.swift

```diff
@@ -148,7 +148,7 @@ class GleanPlumbMessageManager: GleanPlumbMessageManagerProtocol {
                                      object: .homeTabBanner,
                                      value: .messageInteracted,
                                      extras: [TelemetryWrapper.EventExtraKey.messageKey.rawValue: message.id,
-                                              TelemetryWrapper.EventExtraKey.actionUUID.rawValue: UUID()])
+                                              TelemetryWrapper.EventExtraKey.actionUUID.rawValue: UUID().uuidString])
```

Comment:
This should be `uuid` from [line 128][1], no?

[1]: https://github.com/mozilla-mobile/firefox-ios/pull/10729/files#diff-a5eab33472f2dc905aa67ecba0ff5fa02c97215f07b4bbf1a49c5a308ef34f16R128

Reply:
Right - making the changes we discussed on slack!